### ISSUE TITLE
add property to Location to differentiate between international/national/regional?

### ISSUE THREAD

**Issue by StroemPhi** [TIBHannover/ConfIDent_schema#6]
the URIS schema does differentiate between these three and it could be useful for analyzing and filtering events.

**Comment by stephw-tib** [comment]
How do you define this? what if you have speakers from 3 instutions of one region, but attendees but participants from all over the country?
What about a national event that gets joint by 2 participants from abroad? Is it still a national event?

**Comment by StroemPhi** [comment]
My initial idea was to have this defined by the organizers in terms of providing it in the title, as in 12th international anual VIVO Conference. This way we might be able to parse this mostly automaiv from the titles.

**Comment by hauschke** [comment]
What is the current percentage of conferences with "international*" in the title? And do we have examples of something like "3rd National Conference on International Trade", where "international" is part of the subject of the conference?

**Comment by stephw-tib** [comment]
Is every event that does not use "international" in the title not an international event?

**Comment by StroemPhi** [comment]
No, I definitely wouldn't go so far. I would propose this to be an optional property that can be used, if one knows about this. Also implementing it is very low prio atm. And as @hauschke rightly wrote having a title analysis forst might be very helpful to see, how much we could tag with this automatically. I raised this issue, because the Ukrainian RIS has this in their Event model, and I though it might be helpful for filtering.

**Comment by StroemPhi** [comment]
And with regard to the definition of it, I would, after some thought, opt for defining it in accordance with the intended audience. An event that wants to draw an audience from all over the world, is international. An event in which the organizers address a national audience is a national event and so on. So it'S not about the speakers but the audience intended by the organizers, that defines what kind of event it ist.
